lib/fog/vsphere/requests/compute/list_compute_resources.rb

```diff
@@ -0,0 +1,58 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+        def list_compute_resources(filters = { })
```

Comment:
I need to double-check, but I think the other vSphere hashes are returned with String keys.  If that's the case, I recommend changing here for the sake of consistency.